### PR TITLE
User login

### PR DESCRIPTION
**Pull request by bdfairbanks**
It's complete now.

### CODE REVIEW

File: public/timing.js [hunk 1]
```diff
@@ -2,8 +2,6 @@ window.addEventListener("load", function(){
   window.dailyFeed = document.getElementById("dailyFeed");
 
   displayPosts();
-
- 	setInterval(displayPosts, 10000);
```
Comment (MichaelSefranek) [inline public/timing.js:6]: do we need to add this line back in for master? I know we all hate it sufficiently by now.

File: views/index.erb [hunk 2]
```diff
@@ -18,5 +18,29 @@
...
+      <input type="password" name="password" placeholder="Password" required>
+      <input type="submit">
+    </form>
+</div>
+<div>
+ <div id="title"><strong>Sign Up!</strong></div>
+    <form action="/student_signup" method="post">
+      <input type="text" name="username" placeholder="Username" required>
+      <input type= "text" name="github" placeholder="Github Account" required>
+      <input type="password" name="password" placeholder="Password" id = "signup" maxlength= "10" required>
```
Comment (MichaelSefranek) [inline views/index.erb:42]: I thought 10 characters was too little, but for demo purposes this doesn't , matter.

**Review by MichaelSefranek — COMMENTED** [review]
Do we need the setInterval back in here?

File: public/timing.js (hunk 1 above)
Comment (jlocatis) [reply public/timing.js:6]: Yea, it should be in on the master branch right?

File: views/index.erb (hunk 2 above)
Comment (jlocatis) [reply views/index.erb:42]: I think 10 is probably good for now? I guess these days a lot of passwords are longer, need to be. I think 10 is the character limit I set on the admin password too in the database table.

**Pull request merged**